tests/acceptance/integration/g13/VAT_EU_Mode/CheckVATWithUserMultiGroupCest.php

```diff
@@ -0,0 +1,399 @@
+<?php
+/**
+ * @package     redSHOP
+ * @subpackage  Cest
+ * @copyright   Copyright (C) 2008 - 2020 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+use AcceptanceTester\CategoryManagerJoomla3Steps as CategoryManagerJoomla3Steps;
+use AcceptanceTester\ProductManagerJoomla3Steps;
+use AcceptanceTester\TaxGroupSteps;
+use AcceptanceTester\TaxRateSteps;
+use AcceptanceTester\UserManagerJoomla3Steps as UserManagerJoomla3Steps;
+use Configuration\ConfigurationSteps;
+
+/**
+ * Class CheckVATWithUserMultiGroupCest
+ * @since 3.0.2
+ */
+class CheckVATWithUserMultiGroupCest
+{
+	/**
+	 * @var \Faker\Generator
+	 * @since 3.0.2
+	 */
+	protected $faker;
+
+	/**
+	 * @var string
+	 * @since 3.0.2
+	 */
+	protected $categoryName;
+
+	/**
+	 * @var string
+	 * @since 3.0.2
+	 */
+	protected $productName;
+
+	/**
+	 * @var string
+	 * @since 3.0.2
+	 */
+	protected $randomProductNumber;
+
+	/**
+	 * @var int
+	 * @since 3.0.2
+	 */
+	protected $randomProductPrice;
+
+	/**
+	 * @var string
+	 * @since 3.0.2
+	 */
+	protected $taxGroupName;
+
+	/**
+	 * @var array
+	 * @since 3.0.2
+	 */
+	protected $taxRateCompany;
+
+	/**
+	 * @var array
+	 * @since 3.0.2
+	 */
+	protected $taxRatePrivate;
+
+	/**
+	 * @var array
+	 * @since 3.0.2
+	 */
+	protected $taxRateExempt;
+
+	/**
+	 * @var array
+	 * @since 3.0.2
+	 */
+	protected $taxRateRegion;
+
+	/**
+	 * @var array
+	 * @since 3.0.2
+	 */
+	protected $VATSetting;
+
+	/**
+	 * @var array
+	 * @since 3.0.2
+	 */
+	protected $cartSetting;
+
+	/**
+	 * @var array
+	 * @since 3.0.2
+	 */
+	protected $userCompany;
+
+	/**
+	 * @var array
+	 * @since 3.0.2
+	 */
+	protected $userPrivate;
+
+	/**
+	 * @var array
+	 * @since 3.0.2
+	 */
+	protected $userExemptVAT;
+
+	/**
+	 * @var array
+	 * @since 3.0.2
+	 */
+	protected $userRegion;
+
+	/**
+	 * CheckVATWithUserMultiGroupCest constructor.
+	 * @since 3.0.2
+	 */
+	public function __construct()
+	{
+		$this->faker               = Faker\Factory::create();
+		$this->categoryName        = $this->faker->bothify('Category name ?###?');
+		$this->productName         = $this->faker->bothify('Product name ?###?');
+		$this->randomProductNumber = $this->faker->bothify('Product number ?###?');
+		$this->randomProductPrice  = 100;
+
+		// Vat group, vat rate
+		$this->taxGroupName   = $this->faker->bothify('Tax Groups Name ?###?');
+		$this->taxRateCompany = array(
+				'name'         =>  $this->faker->bothify('Tax rate company ?###?'),
+				'group'        => $this->taxGroupName,
+				'shopperGroup' => "Default Company",
+				'country'      => "Denmark",
+				'amount'       => 0.1,
+				'EUCountry'    => 'yes'
```

Comment:
You can delete a space in these lines, so that it looks the same as the others.

Reply:
Let me update tks